apps/emqx_resource/src/emqx_resource_worker.erl

```diff
@@ -879,6 +1029,15 @@ disk_queue_dir(Id, Index) ->
     QDir = binary_to_list(Id) ++ ":" ++ integer_to_list(Index),
```

Comment:
`QDir` here contains special char `:` which is not allowed in Windows.

Reply:
:heavy_check_mark: